src/coreclr/jit/llvm.cpp

```diff
@@ -1798,6 +1796,51 @@ void Llvm::llvmShutdown()
     //    Module.Verify(LLVMVerifierFailureAction.LLVMAbortProcessAction);
 }
 
+void Llvm::populateLlvmArgNums()
+{
+    _shadowStackLclNum = _compiler->lvaGrabTemp(true DEBUGARG("shadowstack"));
+    LclVarDsc* shadowStackVarDsc = _compiler->lvaGetDesc(_shadowStackLclNum);
+    unsigned   nextLlvmArgNum    = 0;
+
+    shadowStackVarDsc->lvLlvmArgNum = nextLlvmArgNum++;
+    shadowStackVarDsc->lvType    = TYP_I_IMPL;
+    shadowStackVarDsc->lvIsParam = true;
+
+    if (_sigInfo.retType == CorInfoType::CORINFO_TYPE_VOID && _info.compRetType != TYP_VOID)
+    {
+        needsReturnStackSlot(_sigInfo.retType, _sigInfo.retTypeClass);
+    }
```

Comment:
Hmm, `needsReturnStackSlot` is a pure function - what does this do then?

Reply:
Lol, yes, I put that in just to have some code to put the breakpoint, I wanted to see if that condition ever happened and forgot to delete it.